packages/application-shell-connectors/src/components/project-extension-image-regex/project-extension-image-regex.spec.tsx

```diff
@@ -83,7 +83,7 @@ describe('rendering', () => {
         {
           request: {
             query: FetchProjectExtensionImageRegex,
-            variables: {
+            context: {
```

Comment:
See release notes to know more about these changes.

FYI: one of the reasons to do that was because the generated graphql types now use an `Exact` object shape for query variables, which obviously does not include request metadata such as `target`, `projectKey`, etc.
This metadata can/should now be passed via the `context` object.